AUTHORS

```diff
@@ -5,3 +5,4 @@
 
 Google Inc.
 Chromium authors.
+Stephan Nordnes Eriksen <stephan-nordnes-eriksen>.
```

Comment:
Is this supposed to be an email?

Reply:
I could not find an email.

Reply:
I am not able to find this person. Any context?

Reply:
Nvm. I see this is a GitHub user.